### PR TITLE
Explain blender shrinkwrap operation

### LINKED ISSUE #9912: [DOC] Adding an example of a new way of fixing intersecting meshes in Blender
Hi, I found a way of nicely fixing intersecting meshes in Blender. I've created a figure to explain it and would like to add it to the [Editing BEM surfaces in Blender tutorial](https://mne.tools/stable/auto_tutorials/forward/80_fix_bem_in_blender.html). The solution is especially useful when you want the correction to follow the curvature of the outer surface (I've hower chosen to use a simpler case for the example).

Please let me know if it is clear enough.

The procedure is the following:
(1) Select the surface that is creating the problem. (2) In _Edit Mode_, press _C_ to use the circle selection tool to select the vertices that are outside.(3-5) In the _Object Data Properties_ tab use the _+_ button to add a _Vertex Group_ and click _Assign_ to assign the current selection to the group. (6-8) In the _Modifiers_ tab go to _Add Modifier_ add a _Shrinkwrap_ modifier and set it to snap _Inside_ with the outer surface as the _Target_ and the _Group_ that you created before as the _Vertex Group_. You can then use the _Offset_ parameter to adjust the distance. (9) In _Object Mode_ click on the downpointing arrow of the _Shrinkwrap_ modifier and click on _Apply_.

example_fix_bem

@wmvanvliet
@mkadwani

### PR DESCRIPTION
**Pull request by wmvanvliet**
#### Reference issue
Example: Fixes #9912.

#### What does this implement/fix?
In the tutorial on how to use Blender to fix BEM meshes, this PR adds a description of the "Shrinkwrap" modifier that was found to be extremely useful for fixing "skull outside of outer skin" errors.

#### Additional information
@ezemikulan kindly provided the text and the screenshot.

### CODE REVIEW

File: tutorials/forward/80_fix_bem_in_blender.py [hunk 1]
```diff
@@ -207,6 +207,26 @@
...
+#
+# (1) Select the surface that is creating the problem. (2) In *Edit Mode*, press
+# ``C`` to use the circle selection tool to select the vertices that are
+# outside.(3-5) In the *Object Data Properties* tab use the ``+`` button to add a
+# *Vertex Group* and click *Assign* to assign the current selection to the group.
+# (6-8) In the *Modifiers* tab go to *Add Modifier* add a *Shrinkwrap* modifier
+# and set it to snap *Inside* with the outer surface as the *Target* and the
+# *Group* that you created before as the *Vertex Group*. You can then use the
+# *Offset* parameter to adjust the distance. (9) In *Object Mode* click on the
+# down-pointing arrow of the *Shrinkwrap* modifier and click on *Apply*.
```
Comment (drammock) [inline tutorials/forward/80_fix_bem_in_blender.py:225]:
```suggestion
# (1) Select the surface that is creating the problem. (2) In *Edit Mode*,
# press :kbd:`C` to use the circle selection tool to select the vertices that
# are outside.(3-5) In the *Object Data Properties* tab use the ``+`` button to
# add a *Vertex Group* and click *Assign* to assign the current selection to
# the group. (6-8) In the *Modifiers* tab go to *Add Modifier* add a
# *Shrinkwrap* modifier and set it to snap *Inside* with the outer surface as
# the *Target* and the *Group* that you created before as the *Vertex Group*.
# You can then use the *Offset* parameter to adjust the distance. (9) In
# *Object Mode* click on the down-pointing arrow of the *Shrinkwrap* modifier
# and click on *Apply*.
```

File: tutorials/forward/80_fix_bem_in_blender.py [hunk 2]
```diff
@@ -214,15 +214,16 @@
...
-# outside.(3-5) In the *Object Data Properties* tab use the ``+`` button to add a
-# *Vertex Group* and click *Assign* to assign the current selection to the group.
-# (6-8) In the *Modifiers* tab go to *Add Modifier* add a *Shrinkwrap* modifier
-# and set it to snap *Inside* with the outer surface as the *Target* and the
-# *Group* that you created before as the *Vertex Group*. You can then use the
-# *Offset* parameter to adjust the distance. (9) In *Object Mode* click on the
-# down-pointing arrow of the *Shrinkwrap* modifier and click on *Apply*.
+# (1) Select the surface that is creating the problem. (2) In *Edit Mode*,
+# press :kbd:`C` to use the circle selection tool to select the vertices that
+# are outside.(3-5) In the *Object Data Properties* tab use the ``+`` button to
```
Comment (larsoner) [inline tutorials/forward/80_fix_bem_in_blender.py:219]:
```suggestion
# are outside. (3-5) In the *Object Data Properties* tab use the ``+`` button to
```

File: tutorials/forward/80_fix_bem_in_blender.py [hunk 3]
```diff
@@ -207,6 +207,27 @@
...
+# ~~~~~~~~~~~~~~~~~~~~
+#
+# A particularly useful operation is the *Shrinkwrap* functionality, that will
+# restrict one surface inside another surface (for example the Skull inside the
+# Outer Skin). Here is how to use it:
+#
+# (1) Select the surface that is creating the problem. (2) In *Edit Mode*,
+# press :kbd:`C` to use the circle selection tool to select the vertices that
+# are outside. (3-5) In the *Object Data Properties* tab use the ``+`` button to
+# add a *Vertex Group* and click *Assign* to assign the current selection to
```
Comment (larsoner) [inline tutorials/forward/80_fix_bem_in_blender.py:220]:
```suggestion
# are outside. (3-5) In the *Object Data Properties* tab use the ``+`` button
# to add a *Vertex Group* and click *Assign* to assign the current selection to
```

**Review by larsoner — APPROVED** [review]

**Review by drammock — APPROVED** [review]

File: tutorials/forward/80_fix_bem_in_blender.py [hunk 4]
```diff
@@ -207,6 +207,27 @@
 # mne.write_head_bem(op.join(head_dir, 'sample-head.fif'), coords, faces,
 #                    overwrite=True)
 
+# Blender editing tips
```
Comment (larsoner) [inline tutorials/forward/80_fix_bem_in_blender.py:210]: Did not render properly

https://37211-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/forward/80_fix_bem_in_blender.html#sphx-glr-auto-tutorials-forward-80-fix-bem-in-blender-py

```suggestion
# %%
# Blender editing tips
```

File: tutorials/forward/80_fix_bem_in_blender.py [hunk 5]
```diff
@@ -207,6 +207,7 @@
 # mne.write_head_bem(op.join(head_dir, 'sample-head.fif'), coords, faces,
 #                    overwrite=True)
 
+# %%
```
Comment (drammock) [inline tutorials/forward/80_fix_bem_in_blender.py:210]: you beat me to it @larsoner

**Comment by larsoner** [comment]
Thanks @wmvanvliet @ezemikulan !

**Comment by wmvanvliet** [comment]
thanks @ezemikulan!

**Comment by ezemikulan** [comment]
Great!

**Pull request merged**